src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -0,0 +1,14 @@
+package com.google.ehub.data;
```

Comment:
Some of the spacing is off in your files. Can you run  `clang-format-path .` to format your files? 

Instructions to set up the formatter: https://github.com/stekolla/sps-clang-format